src/components/dataset-details-page/index.tsx

```diff
@@ -40,6 +40,8 @@ import ExternalLink from '../link-external';
 
 import SC from './styled';
 
+import { Theme } from '../../types';
+
```

Comment:
Kan importere typen + fjerne mellomrom:
```suggestion
import type { Theme } from '../../types';
```